message_definitions/v1.0/planckAero.xml

```diff
@@ -73,9 +73,15 @@
             <entry value="4" name="PLANCK_DECK_SPOOL_EOL_WARNING"/>
             <entry value="5" name="PLANCK_DECK_SPOOL_EOL_ERROR"/>
         </enum>
-	<enum name="PLANCK_CMD_REQ_TYPE"> <!-- Extending with DECK LID commands -->
-            <entry value="5" name="PLANCK_CMD_REQ_DECK_LID_OPEN"/>
-            <entry value="6" name="PLANCK_CMD_REQ_DECK_LID_CLOSE"/>
+        <enum name="PLANCK_CMD_REQ_TYPE"> <!-- Extending with DECK LID commands -->
+            <entry value="5" name="PLANCK_CMD_REQ_DECK_LID_OPEN">
+                <description>Enable Planck stateinfo data</description>
```

Comment:
I'm not sure how this <description> field is supposed to be used, but `Enable Planck stateinfo data` doesn't sound right.